File: Python/FormaAsteriscos.py
class FormaAsteriscos():
    __longitud = None

    def getLongitud(self):
        return self.__longitud

    def setLongitud(self, longitud):
        self.__longitud = longitud

    def dibujarLineaHorizontal(self):
        horizontal = ''
        for i in range(self.__longitud):
            horizontal += "* "
        return horizontal

    def dibujarLineaVertical(self):
        vertical = ''
        for j in range(self.__longitud):
            vertical += "*\n"
        return vertical

    def dibujarEle(self):
        formaEle = ''
        temporal = FormaAsteriscos()
        temporal.setLongitud(self.__longitud - 1)
        formaEle = formaEle + temporal.dibujarLineaVertical()
        temporal.setLongitud(self.__longitud + 1)
        formaEle = formaEle + temporal.dibujarLineaHorizontal()
        return formaEle


def main():
    asteriscos = FormaAsteriscos()
    cantidad = int(input("Ingrese la cantidad de asteriscos: "))
    asteriscos.setLongitud(cantidad)
    print("La linea horizontal queda: ")
    print(asteriscos.dibujarLineaHorizontal())
    print("La linea vertical queda: ")
    print(asteriscos.dibujarLineaVertical())
    print("La forma de ele(L) queda: ")
    print(asteriscos.dibujarEle())


if __name__ == '__main__':
    main()

File: Java/FormaAsteriscos.java
import java.util.Scanner;

public class FormaAsteriscos {
    private int longitud;

    public int getLongitud() {
        return longitud;
    }

    public void setLongitud(int longitud) {
        this.longitud = longitud;
    }

    public String lineaHorizontal() {
        String horizontal = "";
        for (int i = 0; i < longitud; i++) {
            horizontal = horizontal + "* ";
        }
        return horizontal;
    }

    public String lineaVertical() {
        String vertical = "";
        for (int i = 0; i < longitud; i++) {
            vertical = vertical + "*\n";
        }
        return vertical;
    }

    public String lineaEle() {
        String formaEle = "";
        setLongitud(longitud - 1);
        formaEle = formaEle + lineaVertical();
        setLongitud(longitud + 1);
        formaEle = formaEle + lineaHorizontal();
        return formaEle;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        FormaAsteriscos asteriscos = new FormaAsteriscos();

        System.out.println("Ingrese la cantidad de asteriscos: ");
        int cantidad = scanner.nextInt();
        asteriscos.setLongitud(cantidad);
        System.out.println("La linea horizontal queda: ");
        System.out.println(asteriscos.lineaHorizontal());
        System.out.println("\nLa linea vertical queda: ");
        System.out.println(asteriscos.lineaVertical());
        System.out.println("\nLa forma de ele(L) queda: ");
        System.out.println(asteriscos.lineaEle());
    }
}
